proxies/events-graphql-federation/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.5.1](https://github.com/City-of-Helsinki/events-helsinki-monorepo/compare/federation-router-v1.5.0...federation-router-v1.5.1) (2023-11-08)
+
+
+### Bug Fixes
+
+* **tests:** Fix SearchSelect test testing array value/defaultValue ([25e7cd4](https://github.com/City-of-Helsinki/events-helsinki-monorepo/commit/25e7cd44a2246691bd935f6c7342c4bc429ce3e2))
+
```

Comment:
Why is this file included here in the router update?

Reply:
I think the Unified-SEarch has changed, but that change is not included in this release note

Reply:
These Unified-SEarch changes should be mentioned in the release notes https://github.com/City-of-Helsinki/unified-search/pull/115